workflows/resources_test_scripts/demuxafy_test_data.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+set -eo pipefail
+
+
+# ensure that the command below is run from the root of the repository
+REPO_ROOT=$(git rev-parse --show-toplevel)
+cd "$REPO_ROOT"
+
+mkdir -p .cache
+
+# settings
+ID=demuxafy_test_data
+OUT=resources_test/$ID
+
+# https://www.dropbox.com/s/m8u61jn4i1mcktp/TestData4PipelineSmall.tar.gz
+
+
+wget "https://www.dropbox.com/s/m8u61jn4i1mcktp/TestData4PipelineSmall.tar.gz" -O ".cache/TestData4PipelineSmall.tar.gz"
```

Comment:
Is there data that is already present in the openpipelines s3 bucket that we can use for the test? We want to keep the download size as small as possible.